File: loan-repayment-java/src/main/java/com/loan/repayment/exception/ApplicationException.java
package com.loan.repayment.exception;

/**
 *
 * @author Ramachandra
 *
 */
public class ApplicationException extends RuntimeException {

    private static final long serialVersionUID = 1L;

    /**
     * This is Custom Exception.
     *
     * @param message Exception message
     */
    public ApplicationException(final String message) {
        super(message);
    }

}

File: loan-repayment-kotlin/src/main/kotlin/com/loan/repayment/exception/ApplicationException.kt
package com.loan.repayment.exception

/**
 *
 * @author Rama chandra
 */
data class ApplicationException
/**
 * This is Custom Exception.
 *
 * @param message Exception message
 */
(override val message : String) : RuntimeException(message) {
    companion object {
        private const val serialVersionUID = 1L
    }
}